src/main/java/org/venice/beachfront/bfapi/services/JobService.java

```diff
@@ -261,6 +282,41 @@ public void createJobError(Job job, String error) {
 					HttpStatus.INTERNAL_SERVER_ERROR);
 		}
 	}
+	
+	/**
+	 * Gets the detection data as a GPKG archive for a detection based on the Job ID
+	 * 
+	 * @param jobId
+	 *            The ID of the Job
+	 * @return The GPKG archive for the job
+	 */
+	public byte[] getDetectionGeoPackage(String jobId) throws UserException {
+		byte[] geoJsonData = this.getDetectionGeoJson(jobId);		
+		HttpHeaders headers = new HttpHeaders();
+		headers.setContentType(MediaType.APPLICATION_JSON);
+		HttpEntity<byte[]> request = new HttpEntity<byte[]>(geoJsonData, headers);
+		
+		URI geoJsonToGpkgUri = UriComponentsBuilder.newInstance()
+				.scheme(this.GPKG_CONVERTER_PROTOCOL)
+				.host(this.GPKG_CONVERTER_SERVER)
+				.port(this.GPKG_CONVERTER_PORT)
+				.pathSegment("convert")
+				.build().toUri();
+
+		ResponseEntity<byte[]> response;
+		try {
+			response = this.restTemplate.exchange(geoJsonToGpkgUri, HttpMethod.POST, request, byte[].class);
+		} catch (RestClientException ex) {
+			throw new UserException("Error communicating with GeoPackage converter service", ex, HttpStatus.BAD_GATEWAY);
```

Comment:
Filip: I think it makes sense to proxy back the status returned from the converter.  Thoughts?

Reply:
I thought I did that. I must have not pushed up the change. I will make that change.